docs/release-notes.mdx

```diff
@@ -16,6 +16,7 @@ import { TitleCard } from './components/TitleCard';
 
 #### 🐛 Bug Fixes
 - `Select` to have cursor pointer not-allowed
+- `Input` boxed variant label respect host background color
```

Comment:
is it a bug or an enhancement? 

I think the agreement is to put only the most important things into the documentation and keep everything in the changelog. Right @lukahartwig ?

Reply:
Cool, let me know how to proceed here before merge this.

Reply:
If the merge commit has the prefix `fix:` then it will show up as bug fix in the changelog. I think we should use the release notes only for important stuff like features or breaking changes.